mountpoint-s3/src/prefetch/backpressure_controller.rs

```diff
@@ -122,12 +122,13 @@ impl BackpressureController {
     }
 
     // Send an increment read window request to the stream producer
-    async fn increment_read_window<E>(&self, len: usize) -> Result<(), PrefetchReadError<E>> {
+    async fn increment_read_window(&self, len: usize) {
         // This should not block since the channel is unbounded
-        self.read_window_updater
+        let _ = self
```

Comment:
nit: since we are not returning the error, I'd rather use an explicit `if let Err..` to trace instead of `inspect_err`. Not blocking on this though.